src/utils/Utils.java

```diff
@@ -13,9 +16,12 @@
 
 public class Utils {
 
-	public static void createWindow(Stage stage, Object parent, String fxmlLocation, String sceneTitle, Object userData, String cssLocation, Double sceneWidth, Double sceneHeight) {
+	public static void createWindow(Stage stage, Object parent, String fxmlLocation, String sceneTitle, Object userData, String cssLocation, Double sceneWidth, Double sceneHeight, ResourceBundle bundle) {
```

Comment:
Instead of receive the Resource Bundle as a parameter, let's receive a String with that contains the bundle to get. Move all the logic of getting the bundle's to this method.